.travis.yml

```diff
@@ -10,9 +10,6 @@ dist: trusty
 addons:
   chrome: stable
 
-cache:
-  yarn: true
-
```

Comment:
travis/yarn caching doesn't seem to work well with github/master dependencies. I just had this issue:
```bash
yarn
ember test # <-- fails
rm yarn.lock
yarn # (no diff!)
ember test # <-- works
```
removing the local yarn cache did the trick for me. Let's switch this back on once we're doing regular releases (in case we didn't move to the mono-addon until then)